### PR TITLE
README editorial

### PR DESCRIPTION
**Pull request by katmurp**
- Deep edit of README instructions.
- Removing a couple of extraneous files and adding them to gitignore.
- Requesting technical correctness and language reviews from @garyverhaegen-da, @carrielaben-da, and @loiskilberg-da 
- Warning, this is quite chunky.
- No urgency at all.
- Oh, btw, the little TOC won't work until live.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,204 +1,176 @@
...
-direnv: loading ~/daml-projects/docs.daml.com/.envrc
-direnv: using nix
-direnv: export +AR +AR_FOR_TARGET +ARTIFACTORY_USERNAME +ARTIFACTORY_PASSWORD ...
-```
+:warning: The release process pushes new files, overwriting existing ones, but does not at the moment delete anything; i.e. if a file no longer exists in a new version, there will still be a copy of the old version of that file.
 
-The list will be longer, but it is importan that it includes
-`ARTIFACTORY_USERNAME` and `ARTIFACTORY_PASSWORD`. If it doesn't, that means
-your `.envrc.private` file does not properly set those values.
+### Making changes to current or past versions *from 2.0.0 onwards*
```
Comment (garyverhaegen-da) [inline README.md:172]: Note that this is extremely rare.

File: README.md [hunk 2]
```diff
@@ -1,204 +1,176 @@
...
-- `ARTIFACTORY_PASSWORD`: Required to access intermediate build artifacts.
-  Despite the name, this is actually your API key, **not** your password. You
-  can find your API key on [your Artifactory profile page][artifactory], as the
-  first field under "Authentication Settings".
+### Making changes to the next, unreleased version
 
-Here is a made-up example to show the structure:
+1. Make the changes to docs in the Daml or Canton repo, create a PR, and merge into main.
+2. A snapshot is generated every 24 hours that includes the PR.
+3. To get the snapshot details, run the `dep` tool from the root.
```
Comment (garyverhaegen-da) [inline README.md:129]:
```suggestion
3. To get the snapshot details, run the `deps` tool from the root.
```

It actually works from anywhere in the repo (but not outside it).

File: README.md [hunk 3]
```diff
@@ -1,204 +1,176 @@
...
 tar xf docs/workdir/target/html-docs-$(jq -r '.daml' LATEST).tar.gz
 cd html
 python3 -m http.server 8000 --bind 127.0.0.1
 ```
 
-## Setup
+## How to commit changes to the docs
 
-### direnv
+:warning: This repo ignores any pre-2.0.0 docs; those will still be published using their existing release process: tbc what that is!
```
Comment (garyverhaegen-da) [inline README.md:121]: We've not even tried to change the docs for the 1.x site in over a year so unlikely to ever happen.

The process for 1.x is entirely encapsulated in the daml repo; I don't think it should be documented here.

File: README.md [hunk 4]
```diff
@@ -1,204 +1,176 @@
...
 ./download.sh
 ```
 
-## Preview
+### Live preview
 
-For local development, use the `live-preview.sh` script. The files for the
-unified documentation will be in the `docs/workdir/build` folder; changes to
-those files should be immediately reflected.
+Run the `live-preview.sh` script to render a local view of the site. The html files are in `docs/workdir/build`. Edits should be immediately reflected.
```
Comment (garyverhaegen-da) [inline README.md:84]: This seems a but ambiguous to me: you should definitely not edit the HTML files. Perhaps:
```suggestion
Run the `live-preview.sh` script to render a local view of the site. The html files are in `docs/workdir/build`. Edits to `docs/index/` files should be immediately reflected.
```

**Review by garyverhaegen-da — APPROVED** [review]
Thanks!

File: README.md (hunk 1 above)
Comment (katmurp) [reply README.md:172]: OK, cool.

File: .gitignore [hunk 5]
```diff
@@ -1 +1,2 @@
 .envrc.private
+.DS_Store
```
[the remark this answers is not in the document]
Comment (katmurp) [reply .gitignore:2]: thanks for the info here

File: README.md (hunk 3 above)
Comment (katmurp) [reply README.md:121]: OK, I'll add something about reaching out to daml team if it ever comes up.

**Pull request merged**